pkg/apis/faros/v1alpha1/gittrack_types.go

```diff
@@ -17,10 +17,17 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// CredTypeSSH defines a private SSH key credential type
+	CredTypeSSH = "SSH"
```

Comment:
I think we should make these typed strings

Can we make a type alias for this and then call these two as below?

```
type GitTrackDeployKeyType string

const (
  DeployKeyTypeSSH GitTrackDeployKeyType = "SSH"
  DeployKeyTypeOAuthToken GitTrackDeployKeyType = "OAuthToken"
)
```